neuralmonkey/nn/ortho_gru_cell.py

```diff
@@ -5,7 +5,57 @@
 class OrthoGRUCell(tf.contrib.rnn.GRUCell):
     """Classic GRU cell but initialized using random orthogonal matrices."""
 
-    def __call__(self, inputs, state, scope=None):
-        with tf.variable_scope(scope or "OrthoGRUCell") as vscope:
-            vscope.set_initializer(tf.orthogonal_initializer())
-            return super().__call__(inputs, state, vscope)
+    def __init__(self,
+                 num_units,
+                 activation=None,
+                 reuse=None,
+                 bias_initializer=None):
+        tf.contrib.rnn.GRUCell.__init__(
+            self, num_units, activation, reuse, tf.orthogonal_initializer(),
+            bias_initializer)
+
+    def __call__(self, inputs, state, scope="OrthoGRUCell"):
+        return tf.contrib.rnn.GRUCell.__call__(self, inputs, state, scope)
+
+
+# Note that tensorflow does not like when the typing annotations are present.
+class NematusGRUCell(tf.contrib.rnn.GRUCell):
+
```

Comment:
Tady je potřeba docstring, který popíše, jak se to liší od implementace, která je defaultně v TensorFlow.